habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -294,6 +294,16 @@ class BaseVelocityNonCylinderActionConfig(ActionConfig):
     enable_lateral_move: bool = False
     # If the condition of sliding includs the checking of rotation
     enable_rotation_check_for_dyn_slide: bool = True
+    # If we want to do leg animation or not
+    animate_leg: bool = False
+    # Leg animation checkpoint file
+    leg_animation_checkpoint: str = (
+        "data/robots/spot_data/spot_walking_trajectory.csv"
```

Comment:
Where does this come from? Is it hosted on HF?

Reply:
Hi @aclegg3 : If it helps, I found the file present in the below merge request and was able to run the leg animation successfully - 

https://github.com/facebookresearch/habitat-lab/pull/1554